src/overlays/actors/ovl_Obj_Warpstone/z_obj_warpstone.h

```diff
@@ -5,14 +5,30 @@
 
 struct ObjWarpstone;
 
-typedef void (*ObjWarpstoneActionFunc)(struct ObjWarpstone* this, GlobalContext* globalCtx);
+typedef s32 (*ObjWarpstoneActionFunc)(struct ObjWarpstone* this, GlobalContext* globalCtx);
+
+typedef enum {
+    /* 0 */ SEK_MODEL_CLOSED,
+    /* 1 */ SEK_MODEL_OPENED
+} ObjectSekModels;
+
+typedef enum {
+    /* 25 */ OBJ_WARPSTONE_TIMER_OPENED = 25,
+    /* 66 */ OBJ_WARPSTONE_TIMER_DONE = 66
+} ObjWarpstoneTimerCutoffs;
 
 typedef struct ObjWarpstone {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x68];
-    /* 0x01AC */ ObjWarpstoneActionFunc actionFunc;
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ ColliderCylinder collider; 
+    /* 0x1A8 */ u8 isTalking;
+    /* 0x1A9 */ u8 timer /*for statue opening cutscene*/;
+    /* 0x1AA */ u8 modelIndex;
```

Comment:
just a very minor style thing but id personally recommend doing a single line // comment after the semicolon
`/* 0x1A9 */ u8 timer; // for statue opening cutscene`

Reply:
I have been avoiding single line comments because I hate relying on the Microsoft extensions to C89 and would rather adhere to the standard. But since IDO does support them, I guess it's kind of a moot point.

Reply:
yea they are everywhere in both oot and mm already. i wouldnt worry personally

Reply:
fixed in latest push. Marking as resolved.